src/main/java/org/terasology/metalrenegades/population/component/PopulationComponent.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.metalrenegades.population.component;
+
+import org.terasology.entitySystem.Component;
+
+public class PopulationComponent implements Component {
```

Comment:
I think this is somewhat self-explanatory in this case, but I'd like to see some JavaDoc nonetheless 🧐 

```suggestion
/**
 * Track total population information of good, bad, and neutral citizens.
 *
 * The entity this component belongs to defines the scope of the tracked population. 
 * For instance, it may be attached to a world entity for the overall population or a settlement entity for the population of that settlement.
 *
 * The population is usually tracked by the {@link PopulationSystem}.
 */
public class PopulationComponent implements Component {
```

I always hope that this kind of documentation makes it easier for new contributors to understand connections between components and systems, and how all of this works together.